cmd/deploy/wait.go

```diff
@@ -84,8 +86,10 @@ func (dc *DeployCommand) waitForResourcesToBeRunning(ctx context.Context, opts *
 			if err != nil {
 				return err
 			}
-			for _, sfs := range sfsList {
-				if !statefulsets.IsRunning(ctx, opts.Manifest.Namespace, sfs.Name, c) {
+			sfss := make([]v1.StatefulSet, len(sfsList))
+			for i := range sfss {
```

Comment:
Same as above

Reply:
my bad, this was a fix from a deepsource detection. fixed now